Mage.Sets/src/mage/cards/j/JasmineBorealOfTheSeven.java

```diff
@@ -0,0 +1,111 @@
+package mage.cards.j;
+
+import mage.MageInt;
+import mage.MageObject;
+import mage.Mana;
+import mage.abilities.Abilities;
+import mage.abilities.Ability;
+import mage.abilities.SpellAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.combat.CantBeBlockedByCreaturesAllEffect;
+import mage.abilities.keyword.special.JohanVigilanceAbility;
+import mage.abilities.mana.ConditionalColoredManaAbility;
+import mage.abilities.mana.conditional.ConditionalSpellManaBuilder;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterSpell;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicate;
+import mage.filter.predicate.mageobject.NoAbilityPredicate;
+import mage.game.Game;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class JasmineBorealOfTheSeven extends CardImpl {
+
+    private static final FilterSpell spell_filter
+            = new FilterSpell("creature spells with no abilities");
+    private static final FilterCreaturePermanent your_creatures_filter
+            = new FilterCreaturePermanent("creatures you control with no abilities");
+    private static final FilterCreaturePermanent with_abilities_filter
+            = new FilterCreaturePermanent("by creatures with abilities");
+
+    static {
+        spell_filter.add(NoAbilityPredicate.instance);
+        spell_filter.add(CardType.CREATURE.getPredicate());
+        your_creatures_filter.add(NoAbilityPredicate.instance);
+        your_creatures_filter.add(TargetController.YOU.getControllerPredicate());
+        with_abilities_filter.add(new AnyAbilityPredicate());
```

Comment:
Rather than make a new class for this, can you not invert `NoAbilityPredicate`?

Reply:
Good catch. I missed the existence of Not because I was expecting that to be in a separate file when I was looking for a way to invert a predicate.